lib/Value/Matrix.pm

```diff
@@ -6,128 +6,156 @@
 
 =head1 Value::Matrix class
 
+This is the Math Object code for a Matrix.
 
-References:
+=head2 References:
 
-MathObject Matrix methods: L<http://webwork.maa.org/wiki/Matrix_(MathObject_Class)>
-MathObject Contexts: L<http://webwork.maa.org/wiki/Common_Contexts>
-CPAN RealMatrix docs: L<http://search.cpan.org/~leto/Math-MatrixReal-2.09/lib/Math/MatrixReal.pm>
+=over
 
-Allowing Matrices in Fractions:
-L<http://webwork.maa.org/moodle/mod/forum/discuss.php?d=2978>
+=item MathObject Matrix methods: L<http://webwork.maa.org/wiki/Matrix_(MathObject_Class)>
 
-     Context()->parens->set("[" => {formMatrix => 1});
+=item MathObject Contexts: L<http://webwork.maa.org/wiki/Common_Contexts>
 
-Files interacting with Matrices:
+=item CPAN RealMatrix docs: L<http://search.cpan.org/~leto/Math-MatrixReal-2.09/lib/Math/MatrixReal.pm>
 
-L<MatrixReal1>
+=back
 
-L<MatrixReduce.pl>
+=head2 Matrix-Related libraries and macros:
 
-L<Matrix>
+=over
 
-L<MatrixCheckers.pl> -- checking whether vectors form a basis
+=item L<MatrixReal1>
 
-L<MatrixReduce.pl>  -- tools for  row reduction via elementary matrices
+=item L<MatrixReduce.pl>
 
-L<MatrixUnits.pl>   -- Generates unimodular matrices with real entries
+=item L<Matrix>
 
-L<PGmatrixmacros.pl>
+=item L<MatrixCheckers.pl> -- checking whether vectors form a basis
 
-L<PGmorematrixmacros.pl>
+=item L<MatrixReduce.pl>  -- tools for  row reduction via elementary matrices
 
-L<PGnumericalmacros.pl>
+=item L<MatrixUnits.pl>   -- Generates unimodular matrices with real entries
 
-L<tableau.pl>
+=item L<PGmatrixmacros.pl>
 
-quickMatrixEntry.pl
+=item L<PGmorematrixmacros.pl>
 
-L<LinearProgramming.pl>
+=item L<PGnumericalmacros.pl>
 
-Contexts
+=item L<tableau.pl>
 
-	Matrix -- allows students to enter [[3,4],[3,6]]
-	       -- formMatrix =>1 also allows this?
-	Complex-Matrix -- allows complex entries
+=item L<quickMatrixEntry.pl>
 
-Creation methods
+=item L<LinearProgramming.pl>
 
-		 $M1 = Matrix([1,2],[3,4]);
-   		 $M2 = Matrix([5,6],[7,8]);
-    	  $v = Vector(9,10);
-    	  $w = ColumnVector(9,10); # differs in how it is printed
+=back
+
+=head2 Contexts
+
+=over
+
+=item C<Matrix>
+
+Allows students to enter C<[[3,4],[3,6]]>
+
+=item C<Complex-Matrix>
+
+Allows complex entries
+
+=back
+
+
+=head2 Creation of Matrices
+
+Using the C<Matrix>, C<Vector> or C<ColumnVector> methods
+
+Examples:
+
+    $M1 = Matrix([1,2],[3,4]);
+    $M2 = Matrix([5,6],[7,8]);
 
 Commands added in Value::matrix
 
-	Conversion
-		$matrix->values produces [[3,4,5],[1,3,4]] recursive array references of numbers (not MathObjects)
-		$matrix->wwMatrix   produces CPAN MatrixReal1 matrix, used for computation subroutines
+Conversion:
 
-	Information
-		$matrix->dimension:  ARRAY
+    $matrix->value produces [[3,4,5],[1,3,4]] recursive array references of numbers (not MathObjects)
```

Comment:
This seems to be incorrect. (It was incorrect before this PR.) `$matrix->value` produces an actual array, not an array reference. Its entries are either numbers (if `$matrix` was 1D) or are array references. I don't have a suggestion for how to succinctly write that in the POD.